net/raft/raft.go

```diff
@@ -381,55 +379,56 @@ func runTicks(rn raft.Node) {
 
 // Exec proposes the provided instruction and waits for it to be
 // satisfied.
-func (sv *Service) Exec(ctx context.Context, instruction []byte) error {
+func (sv *Service) Exec(ctx context.Context, instruction []byte) (satisfied bool, err error) {
 	prop := proposal{Wctx: randID(), Instruction: instruction}
 	data, err := json.Marshal(prop)
 	if err != nil {
-		return errors.Wrap(err)
+		return satisfied, errors.Wrap(err)
```

Comment:
Nit, these could all say `false` instead. I think that'd be easier to interpret since they will always actually be false.